### PR TITLE
Updated sdk page

### PR DESCRIPTION
**Pull request by bklare**
@sklum @jklontz Suggested changes based on previous converstaions. LMK what you think.

### CODE REVIEW

File: sdk/index.html [hunk 1]
```diff
@@ -61,6 +61,19 @@
 
 <h2>ROC SDK</h2>
 <p class="lead">Find the face you are looking for.</p>
+<h3>At a glance</h3>
+<p>
+The ROC SDK provides software developers and system engineers with state of the art face recognition abilities. The software development kit is accesible througha  C API or a robust set of command line applications. Notable features in the SDK include:
```
Comment (sklum) [inline sdk/index.html]: "througha " -> "through a"

**Comment by sklum** [comment]
One typo.  Other than that, looks great!

**Comment by bklare** [comment]
Thanks for catching. If @jklontz agrees then I will merge.

**Comment by jklontz** [comment]
This reminds me, the other feature we should add is something along the lines of "No network/internet connection required" to help distinguish ourselves against cloud-based offerings. LGTM otherwise!

**Pull request merged**